types/nightmare/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare class Nightmare {
     constructor(options?: Nightmare.IConstructorOptions);
 
     // Interact
-    userAgent(agent: string): Nightmare;
+    // userAgent(agent: string): Nightmare;
```

Comment:
can you delete this or change to the correct case instead?

Reply:
The correct case already exists further down (both were in it). Ill remove this asap